packages/editor/src/components/rich-text/index.native.js

```diff
@@ -520,7 +520,7 @@ export class RichText extends Component {
 
 			this.lastEventCount = undefined; // force a refresh on the native side
 		}
-		let minHeight = 0;
+		let minHeight = styles[ 'editor-rich-text' ].minHeight;
```

Comment:
There's a check for the validity of `style` and `style.minHeight` right after this line. Can we consolidate them somehow?

Reply:
```javascript
let minHeight = styles[ 'editor-rich-text' ].minHeight;
if ( style && style.minHeight ) {
	minHeight = style.minHeight;
}
```

`style` comes from `props`. This seems to be a good way to let the user of RichText modify the default `minHeight`.

Reply:
@hypest did you mean something else by `consolidate` them?
We could set `minHeight` at once but the code might be less readable?

Something like:
```javascript
const minHeight = (style && style.minHeight) || styles[ 'editor-rich-text' ].minHeight;
```

What do you think @marecar3 ?

Reply:
@etoledom looking good to me :)

Reply:
Late to reply but, want to verify that yeah, that looks good to me too!